sdcm/cluster.py

```diff
@@ -1600,7 +1609,7 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster',
         self.log = SDCMAdapter(logger, extra={'prefix': str(self)})
         self.log.info('Init nodes')
         self.nodes = []
-        self.nemesis = []
+        # self.nemesis = []
```

Comment:
why do you leave it commented?

Reply:
the property defined in 2 places BaseCluster and BaseScyllaCluster. I commented on BaseCluster, but forgot to remove before commit. will fix

Reply:
done